code-pushup.config.ts

```diff
@@ -109,7 +109,7 @@ const config: CoreConfig = {
           weight: 1,
         },
       ],
-    },
+    },*/
```

Comment:
Why is the coverage plugin commented out? Leftover from debugging?

Reply:
sry! Forgot it as it was not working locally for some time.